dotCMS/src/integration-test/java/com/dotmarketing/portlets/folders/model/FolderTest.java

```diff
@@ -0,0 +1,65 @@
+package com.dotmarketing.portlets.folders.model;
+
+import static org.junit.Assert.*;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.datagen.FolderDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+
+public class FolderTest {
+
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6325390)